tests/unit/test_artifactory_path.py

```diff
@@ -383,6 +383,7 @@ def test_join_with_multiple_folder_and_artifactory_substr_in_it(self):
         )
 
 
+# todo add unlink test with check when file exists and when not enough permissions
```

Comment:
Let's add it in this PR or remove the comment

Reply:
@allburov 
I will add it, I was waiting other PRs to be merge. Now I can build test on top of them

Reply:
@allburov 
I would say that `responses` handle the lib tests really well, just check tests